### PR TITLE
Use FQCN for property annotations.

### PR DESCRIPTION
**Pull request by dereuromark**
Small doc block fixes.

### CODE REVIEW

**Comment by dereuromark** [comment]
`return $this` (instead of `self`) and the combined setter/getter still kill the IDE in many points. I wonder if they ever solve this :/
But other than that the 3.4 core now looks pretty well, all clickable and almost no "yellow warnings" anymore.

File: src/ORM/Behavior/TreeBehavior.php [hunk 1]
```diff
@@ -731,6 +732,7 @@ protected function _moveDown($node, $number)
                 ->select([$left, $right])
                 ->where(["$parent IS" => $nodeParent])
                 ->where(function ($exp) use ($config, $nodeRight) {
+                    /* @var \Cake\Database\Expression\QueryExpression $exp */
```
Comment (markstory) [inline src/ORM/Behavior/TreeBehavior.php]: Would an IDE be able to figure out a typehint on the closure?

Comment (dereuromark) [inline src/ORM/Behavior/TreeBehavior.php]: No, it has to be insight to have an effect.

Comment (markstory) [inline src/ORM/Behavior/TreeBehavior.php]: Good to know.

**Pull request merged**